esy.json

```diff
@@ -5,19 +5,20 @@
   "repository": "https://github.com/briskml/brisk.git",
   "license": "MIT",
   "scripts": {
+    "fmt": "dune build @fmt --auto-promote",
     "clean": "dune clean",
     "build:core": "esy build -p @brisk/core",
-    "build:mac": "esy build -p @brisk/renderer-macos",
+    "build:mac": "esy build -p @brisk/macos",
     "build:tester_mac": "esy build -p @brisk/tester-macos",
     "test:core": "esy -C -p @brisk/core dune runtest --root . --no-buffer",
-    "test:mac": "esy -C -p @brisk/renderer-macos dune runtest --root . --no-buffer",
+    "test:mac": "esy -C -p @brisk/macos dune runtest --root . --no-buffer",
     "bootstrap:mac": "make -C tester-macos/app bootstrap",
     "run:mac": "esy x make -C tester-macos/app run"
   },
   "dependencies": {
     "@esy-ocaml/reason": ">=3.4.0",
     "@brisk/core": "*",
-    "@brisk/renderer-macos": "*",
```

Comment:
I don't understand this, why does the top package depend on the mac renderer?

Reply:
It’s the new esy monorepos workflow, you might notice the use of ‘link-dev’ in resolutions, which means we pull all the dependencies including devDependencies.